Business Finder/BusinessCell.h

```diff
@@ -0,0 +1,14 @@
+//
+//  BusinessCell.h
+//  Business Finder
+//
+//  Created by Michael Johnson on 2/17/17.
+//  Copyright © 2017 Michael Johnson. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "Business.h"
+
+@interface BusinessCell : UITableViewCell
+@property (strong, nonatomic) Business *business;
```

Comment:
Good point. Now the cell just stores the index, and `NearbyBusinessesTableViewController-prepareForSegue:` retrieves the appropriate `Business` instance and uses it to configure the destination `MapViewController`